addOns/pscanrules/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Added 'DEBUG' to list of suspicious comments.
 - Added custom payload support (via Custom Payloads add-on) to 'Information Disclosure Suspicious Comments' scan rule.
 - Removed suspicious-comments.txt file in favor of payload editing via Custom Payloads add-on.
+- The Information Disclosure Suspicious Comments scan rule now includes example alert functionality for documentation generation purposes (Issue 6119).
+    - It now also has a Alert Tag with a OWASP WSTG reference.
```

Comment:
Since all these entries are about the same scan rule they could be listed under a single main entry.